app/connectors/BaseConnector.scala

```diff
@@ -27,7 +27,7 @@ trait BaseConnector extends HttpErrorFunctions{
     Seq("Content-Type" -> "application/xml")
 
   protected def responseHeaders(): Seq[(String, String)] =
-    Seq((HeaderNames.CONTENT_TYPE, MimeTypes.JSON))
+    Seq("Content-Type" -> "application/json")
```

Comment:
Why are you making these magic strings? HeaderNames and MimeTypes exist explicitly for this purpose.